### ISSUE TITLE
Add plot sampling to simulation capabilities

### ISSUE THREAD

**Issue by LHMarshall** [DistanceDevelopment/dsims#69]
This comes from a suggestion from a design workshop participant, and would be great given increasing use of drones for strip transect sampling. No detection function estimation required, so hopefully not too hard to implement.

**Comment by eosnas** [comment]
@LHMarshall Yes, please! I just wrote some code to approximate this using the available functions in dsims. I just specified a uniform key function, and then fit a half-normal during the analysis step. Seems the detection rate was ~98% for some reps, so this was close enough for my purpose. Not sure if I was the one to suggest this when I took the class, but my current application is aerial photos from a plane (someday a drone) along a transect.
